docsource/developers.rst

```diff
@@ -27,6 +27,144 @@ To compile, simply run ``make compile``. The
 `kb-sdk <https://github.com/kbase/kb_sdk>`_ executable must be in the system
 path.
 
+Deploying the Workspace Service locally
+----------------------------------------
+These instructions are known to work on Ubuntu 16.04 LTS.
+
+1. Install the dependencies pymongo v2.8, GlassFish v3.1.*, mongodb >=v2.6.*, kb-sdk and the KBase Jars directory.
```

Comment:
ant and java8 are also dependencies, forgot again, sorry

Reply:
Also are you sure anything other than 3.1.2.2 will work? I seem to remember trying 3.1.1.x and failing, but I could be wrong... but to be safe I'd just spec 3.1.2.2 exactly.